go.mod

```diff
@@ -99,6 +99,7 @@ require (
 	github.com/hashicorp/yamux v0.1.1 // indirect
 	github.com/imdario/mergo v0.3.13 // indirect
 	github.com/inconshreveable/mousetrap v1.0.1 // indirect
+	github.com/influxdata/influxdb-client-go v1.4.0 // indirect
```

Comment:
You can also update `github.com/influxdata/influxdb-client-go/v2` to ` v2.12.3`.